src/index.js

```diff
@@ -40,36 +41,49 @@ export { default as Utils } from './utils';
  * - MDN: [HTMLTemplateElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTemplateElement)
  */
 
-/*
- * Register element definitions with the Custom Element registry.
- */
-function _defineElements () {
-    for (let element in Elements) {
-        Elements[element].$define();
-    }
-}
+const HelixUI = {
+    Elements,
+    Utils,
+    VERSION,
 
-/**
- * Initialize HelixUI when Web Components are ready.
- */
-export function initialize () {
-    if (window.WebComponents) {
-        // Polyfill detected
-        if (window.WebComponents.ready) {
-            // polyfill already finished loading, initialize immediately
-            _defineElements();
+    /**
+     * Initialize HelixUI when Web Components are ready.
+     * (synchronous)
+     *
+     * @param {Function?} cb - callback function to execute once HelixUI is fully initialized.
+     */
+    initialize (cb) {
```

Comment:
Added callback support to `initialize()` in order to provide consumers a way of tapping into the initialization timings.

TODO: need to verify that it still works if you call `initialize()` without a callback. (maybe default to a noop?)

Reply:
TODO: need to verify that it still works with a non-function argument.

Reply:
How difficult would it be to support callbacks (synchronous) _and_ promise output (asynchronous)?

```javascript
// Synchronous Execution
HelixUI.initialize( () => {
    app.$mount('#app');
});

// Asynchronous Execution
(async function () {
    await HelixUI.initialize();
    app.$mount('#app');
})();

// or
HelixUI.initialize().then( () => {
    app.$mount('#app');
});
```